workflow/full-installertest-flow.groovy

```diff
@@ -0,0 +1,66 @@
+// Full installer test flow, in one file
+// You must parameterize the build with: 
+
+// @stringparameter dockerLabel (node label for docker nodes)
+// @stringparameter packagingBranch - branch to use in packaging build
+
+// **ARTIFACTS URLS**
+// Note: you can use an artifact archived in a Job build by an artifact:// URL
+//  Ex: 'artifact://full/path/to/job/buildNr#artifact.ext'
+// @stringparameter debfile URL to Debian package
+// @stringparameter rpmfile URL to CentOS/RHEL RPM package
+// @stringparameter susefile URL to SUSE RPM package
+// @stringparameter testLibraryBranch branchname to load the installertests workflow lib from
+
+// Optional build parameters
+// @stringparameter (optional) port - port number to use in testing jenkins (default 8080)
+// @stringparameter (optional) artifact (jenkins artifactname, defaults to 'jenkins')
+
+// Basic parameters
+String dockerLabel = 'ubuntu-ope01'
+String packagingBranch = 'feature-docker-test-env-fixes'
+String artifactname = (artifact == null ) ? 'jenkins' : artifact
+String jenkinsPort = (port == null ) ? '8080' : "$port"
+
+// Set up
+String scriptPath = 'packaging-docker/installtests'
+String checkCmd = "sudo $scriptPath/service-check.sh $artifactname $jenkinsPort"
+
+// Core tests represent the basic supported linuxes, extended tests build out coverage further
+def coreTests = []
+def extendedTests = []
+coreTests[0]=["sudo-ubuntu:14.04",  ["sudo $scriptPath/debian.sh installers/deb/*.deb", checkCmd]]
```

Comment:
I did some digging and haven't really found a better way to to this (or a best practice). 
AFAICT what I'm doing here with docker containers is a new (or niche) approach without a defined best practice.

Docker registries do support namespacing, but it's generally mapped to username/org.  For example, I would be pushing personal stuff to svanoort/myimagname:tag.    One alternative is to do baseimage:sudo-version -- but I feel like that's less accurate for what we're doing here (and allows for confusion with normal base images, since the tag can be omitted).

So basically... easy to change if we find a better way :-)